### PR TITLE
add caper

### PR DESCRIPTION
**Pull request by leepc12**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/caper/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  skip: True  # [win]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
```
Comment (mbargull) [inline recipes/caper/meta.yaml:22]: For non-`noarch: python` packages we have to resort to the `[py<36]` selector for skipping the build.
```suggestion
  skip: True  # [py<36 or win]
  number: 0
  script: "{{ PYTHON }} -m pip install . -vv"

requirements:
  host:
    - python
    - pip
  run:
    - python
```

**Comment by mbargull** [comment]
> For recipes/caper:
> - Recipe with the same name exists in bioconda: please discuss with @conda-forge/bioconda-recipes.

Can you open a PR (referring to this PR) on bioconda-recipes to remove it from there?

**Comment by leepc12** [comment]
> > For recipes/caper:
> > 
> > * Recipe with the same name exists in bioconda: please discuss with @conda-forge/bioconda-recipes.
> 
> Can you open a PR (referring to this PR) on bioconda-recipes to remove it from there?

Sure, I am trying to migrate caper from bioconda to conda-forge.

**Comment by leepc12** [comment]
@mbargull: I actually found [your comment](https://github.com/bioconda/bioconda-recipes/issues/20445#issuecomment-589537478) that doesn't recommend removal of a package from bioconda. Can I just keep old versions there?

**Comment by mbargull** [comment]
> Sure, I am trying to migrate caper from bioconda to conda-forge.

Yep, I saw that you were the one that contributed the package to Bioconda and as such omitted my usual "explain what and why what should go to either channel" stanza ;).

> @mbargull: I actually found [your comment](https://github.com/bioconda/bioconda-recipes/issues/20445#issuecomment-589537478) that doesn't recommend removal of a package from bioconda. Can I just keep old versions there?

Yep, that comment is still valid -- however, there is the distinction between recipes and the actually built packages. We want to keep the existing builds on `bioconda`, but we can remove the recipe over there. That way new build will only come from `conda-forge` and we/you don't have to maintain recipes in multiple places :).

**Comment by leepc12** [comment]
@mbargull, can you take a quick look at this PR? Things are already done on the `bioconda` side and I think it's time to get this PR approved and merged?

**Comment by leepc12** [comment]
@mbargull : Sorry bothering you but is there anyone that can approve this PR? I really need to get my package registered to `conda-forge`.

File: recipes/caper/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 33a963c47f9b43c6070c81cd80a40b27df3c00ccc1f7078045a84bb0a7ea8f59 
+
+build:
+  skip: True  # [py<36 or win]
```
Comment (dopplershift) [inline recipes/caper/meta.yaml:13]: Can you go back to `noarch: python` here? We can ignore the windows build failure for that case. I'm assuming it fails due to a package being unavailable on windows?

Comment (leepc12) [reply recipes/caper/meta.yaml:14]: This package is not supposed to work on Windows. I added `noarch: python` back but got linting errors. How can we ignore Windows build?

Comment (dopplershift) [reply recipes/caper/meta.yaml:14]: Oh. Well, if it's outright broken on Windows, then it shouldn't be noarch and you can take that back out. Apologies for the confusion.

Comment (leepc12) [reply recipes/caper/meta.yaml:13]: @dopplershift: Thanks, I just removed the `noarch` stuff.

**Review by dopplershift — APPROVED** [review]

**Pull request merged**